Tools/build/compute-changes.py

```diff
@@ -16,7 +16,7 @@
 
 TYPE_CHECKING = False
 if TYPE_CHECKING:
-    from collections.abc import Set
+    from collections.abc import Set as AbstractSet
```

Comment:
Why the rename?

Reply:
This is a ruff suggestion: https://docs.astral.sh/ruff/rules/unaliased-collections-abc-set-import/#why-is-this-bad.

Reply:
I'd suppress this.